src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,20 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    void insertDrivers(Car car);
+
+    Car get(Long id);

```

Comment:
It's good practice to return Optionals from `get` method from DAO layer

Reply:
done